join.spec.md

```diff
@@ -23,17 +23,17 @@ graph LR
 This document defines schema elements for describing [core schemas](/core/v0.1) which **join** multiple **subgraph** schemas into a single **supergraph** schema.
 
 This specification provides machinery to:
```

Comment:
This list is written like it's exhaustive but doesn't mention `@join__type` or `@join__owner`

Reply:
Fixed on my branch.